cli.js

```diff
@@ -1,13 +1,26 @@
 #!/usr/bin/env node
 
 var chalk = require("chalk");
-var lib = require(".");
+var argv = require("yargs").argv;
+var Changelog = require(".").Changelog;
+var ConfigurationError = require(".").ConfigurationError;
 
-var Changelog = lib.Changelog;
-var ConfigurationError = lib.ConfigurationError;
+if (argv.help) {
```

Comment:
Ah that's perfect! For some reasons I didn't go through the entire readme they have. I'll change that, thanks!